File: nid-agent-node/demo/utils/deepseek.mjs
import OpenAI from 'openai'
import { configDotenv  } from 'dotenv';
configDotenv();
console.log(process.env.DEEPSEEK_API_KEY,process.env.DEEPSEEK_BASE_URL)
export const deepseek = new OpenAI({
    apiKey: process.env.DEEPSEEK_API_KEY,
    baseURL: process.env.DEEPSEEK_BASE_URL,
})

File: ai/py_weapp/yu-ai-learn/backend/app/llm/deepseek.py
from app.core.config import settings


def create_deepseek_llm():
    if not settings.deepseek_api_key:
        raise RuntimeError("DEEPSEEK_API_KEY 未配置")
    try:
        from langchain_openai import ChatOpenAI
    except ImportError as exc:
        raise RuntimeError("请安装可选依赖 langchain-openai") from exc
    return ChatOpenAI(model=settings.deepseek_model, base_url=settings.deepseek_base_url, api_key=settings.deepseek_api_key, temperature=0.4)
